invenio_cli/commands/translations.py

```diff
@@ -0,0 +1,134 @@
+        output_file = self.cli_config.get_project_dir()/Path("translations/messages.pot")
+        input_dirs = self.cli_config.get_project_dir()
+
+        cmd = self.CMD_PREFIX + [
+            "pybabel",
+            "extract",
+            f"--mapping-file={mapping_file}",
+            f"--output-file={output_file}",
+            f"--input-dirs={input_dirs}",
+            f"--msgid-bugs-address={msgid_bugs_address}",
+            f"--copyright-holder={copyright_holder}",
+            f"--add-comments={add_comments}",
+        ]
+
+        return [
+            CommandStep(
+                cmd=cmd,
+                env={"PIPENV_VERBOSITY": "-1"},
+                message=f"Extracting i18n messages from {input_dirs}...",
+            )
+        ]
+
+    def init(self, locale):
+        """initialize a new language catalog."""
+
+        output_dir = self.cli_config.get_project_dir()/Path("translations/")
+        input_file = self.cli_config.get_project_dir()/Path("translations/messages.pot")
+
+        cmd = self.CMD_PREFIX + [
+            "pybabel",
+            "init",
+            f"--output-dir={output_dir}",
+            f"--input-file={input_file}",
+            f"--locale={locale}",
+        ]
+
+        return [
+            CommandStep(
+                cmd=cmd,
+                env={"PIPENV_VERBOSITY": "-1"},
+                message=f"Initializing message catalog for {locale}...",
+            )
+        ]
+
+
+    def update(self):
+        """Update the message catalog."""
+
+        output_dir = self.cli_config.get_project_dir()/Path("translations/")
+        input_file = self.cli_config.get_project_dir()/Path("translations/messages.pot")
+
+        cmd = self.CMD_PREFIX + [
+            "pybabel",
+            "update",
+            f"--output-dir={output_dir}",
+            f"--input-file={input_file}",
+        ]
+
+        return [
+            CommandStep(
+                cmd=cmd,
+                env={"PIPENV_VERBOSITY": "-1"},
+                message=f"Updating message catalog...",
+            )
+        ]
+
+    def compile(self, fuzzy=False):
+        """Compile the message catalog."""
+
+        translation_folder = Path("translations/")
+        dir = self.cli_config.get_project_dir()/translation_folder
+
+        cmd = self.CMD_PREFIX + [
+            "pybabel",
+            "compile",
+            f"--directory={dir}",
+        ]
+
+        if fuzzy:
+            cmd.append("--use-fuzzy")
+
+        return [
+            CommandStep(
+                cmd=cmd,
+                env={"PIPENV_VERBOSITY": "-1"},
+                message=f"Compiling message catalog...",
+            ),
+            FunctionStep(
+                func=self.symlink_translations,
+                args={"target": translation_folder},
+                message="Symlinking translations...",
+            ),
+        ]
+
+    def symlink_translations(self, target):
```

Comment:
This function is duplicated code form some other cmd class (install, and maybe more)